modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -14,10 +14,15 @@ Installing {product-title} on a single node alleviates some of the requirements
 .Hardware requirements
 [options="header"]
 |====
-|Profile|CPU|Memory|Storage
-|Minimum|8 CPU cores|32GB of RAM| 120GB
+|Profile|vCPU|Memory|Storage
+|Minimum|8 vCPU cores|32GB of RAM| 120GB
 |====
 +
+[NOTE]
+====
+One vCPU is equivalent to one physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled. When enabled, use the following formula to calculate the corresponding ratio: (threads per core × cores) × sockets = vCPUs.
```

Comment:
Might be worth putting on a new line for scan-/read-ability (remove punctuation if so):

```
... the corresponding ratio:

(threads per core × cores) × sockets = vCPUs
```